wasm/massa_tools/include_base64.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+const file = process.argv[2];
+const include_bytes_regex = /include_base64\(["']+([\.a-z_\-\/\\]*)["']+\)[;]+/i;
```

Comment:
Did you find this somewhere? Maybe include the link to the source so we know it's good? :)

Reply:
Hm it's my own regex :rofl:

Reply:
if you want I can add some unit test

Reply:
ok, thanks